### PR TITLE
[File Explorer Source Control Integration] Add subheading and open folder in File Explorer functionality

### PR DESCRIPTION
**Pull request by ssparach**
## Summary of the pull request
UX changes

## Detailed description of the pull request / Additional comments
This PR adds a subheading that specifies the source control extension the repository path is registered to (if any). This PR also adds a menu option to open the repository root path in File Explorer. 

image

## Validation steps performed
Build of msix package
Validated UX functionality

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml [hunk 1]
```diff
@@ -29,6 +29,7 @@
                 <DataTemplate>
                     <ctControls:SettingsCard 
                         Header="{Binding RepositoryRootPath}"
+                        Description ="{Binding SourceControlProviderDisplayName}"
```
Comment (krschau) [inline tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml:32]: Nit
```suggestion
                        Description="{Binding SourceControlProviderDisplayName}"
```

File: tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs [hunk 2]
```diff
@@ -42,4 +46,28 @@ public void AssignSourceControlProviderButton_Click(object sender, RoutedEventAr
             ViewModel.AssignSourceControlProviderToRepository(menuItem.Text, repoInfo.RepositoryRootPath);
         }
     }
+
+    public void OpenFolderInFileExplorer_Click(object sender, RoutedEventArgs e)
+    {
+        // Extract relevant data from view and give to view model for assign
```
Comment (krschau) [inline tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs:52]: Remove comment leftover from copy/paste

File: tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs [hunk 3]
```diff
@@ -42,4 +46,27 @@ public void AssignSourceControlProviderButton_Click(object sender, RoutedEventAr
             ViewModel.AssignSourceControlProviderToRepository(menuItem.Text, repoInfo.RepositoryRootPath);
         }
     }
+
+    public void OpenFolderInFileExplorer_Click(object sender, RoutedEventArgs e)
+    {
+        MenuFlyoutItem menuItem = (MenuFlyoutItem)sender;
+        if (menuItem.DataContext is RepositoryInformation repoInfo)
```
Comment (krschau) [inline tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs:53]: I think the first line will crash if it's not a MenuFlyoutItem (I know it currently always would be, but...)
```suggestion
        MenuFlyoutItem menuItem = sender as MenuFlyoutItem;
        if (menuItem?.DataContext is RepositoryInformation repoInfo)
```

**Review by krschau — APPROVED** [review]

File: tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs (hunk 3 above)
Comment (ssparach) [reply tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs]: Incorporated suggestion in latest. Thanks!

**Review by dhoehna — APPROVED** [review]

**Pull request merged**